src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -341,7 +341,7 @@ function ChildReconciler(shouldClone, shouldTrackSideEffects) {
     } else {
       // Update
       const existing = useFiber(current, priority);
-      existing.pendingProps = portal.children;
+      existing.pendingProps = portal;
```

Comment:
An alternate solution could be `portal.children || []`.

This is also a problem with HostRoot so we should choose a consistent strategy.

https://github.com/facebook/react/blob/master/src/renderers/shared/fiber/ReactFiberReconciler.js#L163

However, I'm also considering using an effect tag to indicate the lack or existence of pendingProps instead.